app/controllers/planning_applications/review/tasks_controller.rb

```diff
@@ -5,12 +5,24 @@ module Review
     class TasksController < AuthenticationController
       before_action :set_planning_application
       before_action :ensure_user_is_reviewer
+      before_action :set_condition_set
+      before_action :set_condition_set_review
 
       def index
         respond_to do |format|
           format.html
         end
       end
+
+      private
+
+      def set_condition_set
+        @condition_set = @planning_application.condition_set || @planning_application.create_condition_set!
+      end
+
+      def set_condition_set_review
+        @condition_set_review = @condition_set.review || @condition_set.create_review!
+      end
```

Comment:
Yes, I think it would be good to avoid a situation like we had with consultations, where it was possible for the object not to have been created when already trying to display the status of it.

Reply:
Yes, I think it would be good to avoid a situation like we had with consultations, where it was possible for the object not to have been created when already trying to display the status of it.

(comment moved to correct place 🤦🏻‍♂️)